### DISCUSSION TITLE
How can I reset the super admin password?

### DISCUSSION THREAD

**Discussion by Alik2015** [OrchardCMS/OrchardCore#14229] [General]
I am using the latest orchard cms wth a decoupled razor appliction.

How can I reset the super user password which is forgotten/lost and a real email address was never used?

**Comment by SzymonSel** [comment]
Do you have access to the DB tables?

**Comment by Alik2015** [comment]
DB yes sqllite. Which table? On 31 Aug 2023 20:27, Szymon Seliga ***@***.***> wrote:
Do you have access to the DB tables?

**Comment by SzymonSel** [comment]
In the `Document` table look for your admin user (column `Type` should have a value of `OrchardCore.Users.Models.User, [...]`)

The in column `Content` change the JSON property `PasswordHash` to a value of a password you know. To generate sucha hash, you can create a new user and copy the hash. The user can be created in this or any other OrchardCore app. If user registration is possible, you can also edit this user and set it's role to "Administrator" => `"RoleNames": ["Administrator"]`

**Comment by Alik2015** [comment]
Great appreciate it. On 31 Aug 2023 21:24, Szymon Seliga ***@***.***> wrote:
In the Document table look for your admin user (column Type should have a value of OrchardCore.Users.Models.User, [...])
The in column Content change the JSON property PasswordHash to a value of a password you know. To generate sucha hash, you can create a new user and copy the hash. The user can be created in this or any other OrchardCore app. If user registration is possible, you can also edit this user and set it's role to "Administrator" => "RoleNames": ["Administrator"]
